src/daemon/krillserver.rs

```diff
@@ -180,9 +180,9 @@ impl KrillServer {
 
         let pubserver = {
             if mode.pubd_enabled() {
-                Some(PubServer::build(config.clone(), signer.clone(), &system_actor)?)
+                Some(PubServer::build(config.clone(), signer.clone())?)
```

Comment:
Why is actor no longer passed here?

Reply:
Ah, it was event based before and so an actor was propagated to log as the initiator of the event.